sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/circuit/operator/DBSPStreamAggregateOperator.java

```diff
@@ -51,7 +52,16 @@ public DBSPStreamAggregateOperator(CalciteRelNode node,
         Utilities.enforce(aggregateList == null || !aggregateList.isLinear());
         Utilities.enforce(aggregateList == null ||
                 aggregateList.rowVar.getType().sameType(input.getOutputIndexedZSetType().elementType.ref()));
+        Utilities.enforce(aggregateList == null ||
+                aggregateList.getType().to(DBSPTypeFunction.class).resultType.sameType(outputType.elementType));
         Utilities.enforce(input.getOutputIndexedZSetType().keyType.sameType(outputType.keyType));
+        /*
```

Comment:
maybe you can delete it

Reply:
I want to eventually fix it, but I will do it in a separate PR